cli/cmd/policy.go

```diff
@@ -87,29 +95,79 @@ server.`,
 			}
 			defer conn.Close()
 
-			client := pb.NewInboundServerPoliciesClient(conn)
+			elems := strings.Split(args[0], "/")
+
+			if len(elems) == 1 {
+				return errors.New("resource type and name are required")
```

Comment:
should we maybe mention in the help text that the resource must be given in the form of `<type>/<name>`? in particular i notice that you can't currently say 
```console
$ linkerd policy -n foo pod bar-whatever 9000
```
the way you can say
```console
$ kubectl get -n foo pod bar-whatever
```
and it might be worth having the help text mention that?

Reply:
Resource is shown as a single argument in the usage and both of the examples use the `/` format.  I think that's probably good enough

```
Usage:
  linkerd diagnostics policy [flags] resource port

Examples:
  # get the inbound policy for pod emoji-6d66d87995-bvrnn on port 8080
  linkerd diagnostics policy -n emojivoto po/emoji-6d66d87995-bvrnn 8080

  # get the outbound policy for Service emoji-svc on port 8080
  linkerd diagnostics policy -n emojivoto svc/emoji-svc 8080